verilog/parser/verilog.y

```diff
@@ -4598,9 +4598,10 @@ expr_primary_parens
   ;
 expr_primary_braces
   : '{' '}'
-    { $$ = MakeTaggedNode(N::kExpression, MakeBraceGroup($1, nullptr, $2)); }
+    { $$ = MakeTaggedNode(N::kExpression,
+                          MakeTaggedNode(N::kConcatenationExpression, $1, nullptr, $2)); }
```

Comment:
It's not immediately clear to me whether this case of `{}` should be considered a concatenation expression.  I've also seen it be used to refer to an empty queue.

Reply:
Reverted, thanks for spotting that. I don't think we need to distinguish that here as there are no items to format.